### PR TITLE
Move functional interfaces to com.hazelcast.function

### PR DESCRIPTION
**Pull request by cangencer**
EE: https://github.com/hazelcast/hazelcast-enterprise/pull/3251

### CODE REVIEW

**Comment by mmedenjak** [comment]
Needs EE PR

File: hazelcast/src/main/java/com/hazelcast/internal/util/function/LongLongConsumer.java [hunk 1]
```diff
@@ -1,30 +0,0 @@
...
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.hazelcast.internal.util.function;
-
-/**
- * This is a (long,long) primitive specialisation of a BiConsumer
- */
-public interface LongLongConsumer {
```
Comment (mmedenjak) [inline hazelcast/src/main/java/com/hazelcast/internal/util/function/LongLongConsumer.java]: I see this class is still used in EE - https://github.com/hazelcast/hazelcast-enterprise/blob/de2deb045652ff377ace40e573967843052bff8e/hazelcast-enterprise-native-memory/src/main/java/com/hazelcast/internal/memory/StandardMemoryManager.java#L13. 

Can you revert this change?

**Comment by cangencer** [comment]
> Needs EE PR

Doesn't seem any usage in EE. I've reverted the `LongLongConsumer` change.

**Comment by cangencer** [comment]
Enterprise PR: https://github.com/hazelcast/hazelcast-enterprise/pull/3251

**Comment by cangencer** [comment]
run-lab-run
